packages/sdk/src/services/invitation-service.ts

```diff
@@ -27,9 +28,9 @@ export class InvitationService extends Service<InvitationApi> {
    * @param keypairId Use this public key in the new Connection. This is a Keystore Keypair.id (not external_id).
    * Throws an error if the key pair does not exist.
    */
-  public async list(credentials: IVaultToken & IKeystoreToken & IDEK & IKEK): Promise<{
-    invitations: Invitation[];
-  }> {
+  public async list(
+    credentials: IVaultToken & IKeystoreToken & IDEK & IKEK
+  ): Promise<InvitationsResponse> {
```

Comment:
@gongte1937 could you please also update invitation list tests
https://github.com/Meeco/js-sdk/blob/develop/packages/sdk/test/services/invitation-service.test.ts#L41